opencti-platform/opencti-front/src/private/components/common/containers/ContainerHeader.jsx

```diff
@@ -898,7 +899,7 @@ const ContainerHeader = (props) => {
               <StixCoreObjectFileExport
                 id={container.id}
                 type={container.entity_type}
-                redirectToContent={true}
+                redirectToContent={!!redirectToContent}
```

Comment:
If it's a boolean before, and after, need to use `!!` ?

Reply:
There is uncertainty about the value we want to pass here, as it will not always be "true". For example, in the case of 'note' and 'observed_data', we want the value of 'redirectToContent' to remain "false".